Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,178 @@
+description: Integration to fetch and update symantec DLP Incidents
+detaileddescription: Symantec DLP integration uses SOAP API to communicate with Enforce
+  server to fetch and update  incident Data. uses Zeep package for SOAP Api
+configuration:
+- display: 'credentials '
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: server-ip
```

Comment:
display of params should be nice text with capital case (e.g Server IP)
Is it ip here or `Server URL`?